vite.config.ts

```diff
@@ -1,36 +1,36 @@
 /// <reference types="vitest" />
-import { defineConfig } from 'vite';
```

Comment:
This just a file that hasn't been formatted?

Reply:
It looks like most of the changes are formatting related. 

However, there is an actual change on line 11 were I've added tsx (`(x)?`) to the include list:
```javascript
include: ["**/.specs/**/*.+(spec).ts(x)?"],
```

Reply:
Sorry - I should have highlighted this in the description!